docs/ServerAPI/csharp/Vehicle.md

```diff
@@ -2,37 +2,64 @@
 
 ### Properties
 
-| Property Name       | Type         | Description |
-| ------------------- | ------------ | ----------- |
-| CustomTires         | `bool`       |             |
-| DashboardColor      | `byte`       |             |
-| DirtLevel           | `byte`       |             |
-| Driver              | `IPlayer`    |             |
-| InteriorColor       | `byte`       |             |
-| IsPrimaryColorRgb   | `bool`       |             |
-| IsSecondaryColorRgb | `bool`       |             |
-| ModKit              | `byte`       |             |
-| NeonColor           | `Rgba`       |             |
-| NumberPlateIndex    | `uint`       |             |
-| NumberPlateText     | `string`     |             |
-| PearlColor          | `byte`       |             |
-| PrimaryColor        | `byte`       |             |
-| PrimaryColorRgb     | `Rgba`       |             |
-| SecondaryColor      | `byte`       |             |
-| SecondaryColorRgb   | `Rgba`       |             |
-| SpecialDarkness     | `byte`       |             |
-| TireSmokeColor      | `Rgba`       |             |
-| WheelColor          | `byte`       |             |
-| WheelType           | `byte`       |             |
-| WheelVariation      | `byte`       |             |
-| WindowTint          | `byte`       |             |
-| Type                | `EntityType` |             |
-| Dimension           | `ushort`     |             |
-| Exists              | `bool`       |             |
-| Id                  | `ushort`     |             |
-| NativePointer       | `IntPtr`     |             |
-| Position            | `Position`   |             |
-| Rotation            | `Rotation`   |             |
+| Property Name       | Type         | Access      | Description |
+| ------------------- | ------------ | ----------- |             |
+| Position            | `Position`   | get, set    |             |
+| Rotation            | `Rotation`   | get, set    |             |
+| Type                | `EntityType` | get         |             |
+| Dimension           | `ushort`     | get, set    |             |
```

Comment:
dimension is a short https://github.com/FabianTerhorst/coreclr-module/blob/master/api/AltV.Net/Elements/Entities/IWorldObject.cs#L17